Sources/App/Commands/TriggerBuilds.swift

```diff
@@ -240,10 +240,21 @@ struct BuildPair: Equatable, Hashable {
 struct BuildTriggerInfo: Equatable {
     var versionId: Version.Id
     var pairs: Set<BuildPair>
-
-    init(_ versionId: Version.Id, _ pairs: Set<BuildPair>) {
+    // non-essential fields, used for logging
+    var packageId: Package.Id?
+    var packageName: String?
+    var reference: Reference?
```

Comment:
I'm not in love having to tack on these properties used only for logging but there's no way to do this that I can see that'd be even worse.